src/wing/CCPACSWingSection.h

```diff
@@ -37,32 +37,70 @@ class CCPACSWingSection : public generated::CPACSWingSection
 public:
     TIGL_EXPORT CCPACSWingSection(CCPACSWingSections* parent, CTiglUIDManager* uidMgr);
 
-    // Get element count for this section
+    /**
+    * @brief Get element count for this section.
+    * While CPACS supports several elements per section, a section in TiGL must contain exactly one element
+    * @return int
+    */
     TIGL_EXPORT int GetSectionElementCount() const;
 
-    // Get element for a given index
+    /**
+    * @brief Get element for a given index
+    * @return CCPACSWingSectionElement
+    */
     TIGL_EXPORT CCPACSWingSectionElement& GetSectionElement(int index);
     TIGL_EXPORT const CCPACSWingSectionElement& GetSectionElement(int index) const;
 
-    // Gets the section transformation
+    /**
+    * @brief Gets the section transformation
+    * @return CTiglTransformation
+    */
     TIGL_EXPORT CTiglTransformation GetSectionTransformation() const;
 
-    // Gets the section translation
+    /**
+    * @brief Returns the translation of a wing section in CPACS-coordinates
```

Comment:
What are "CPACS-coordinates"? I don't think this is a defined term and people might struggle looking this up.

Reply:
It seems to be a defined term in CPACS, there is a lot to read about it in the documentation for the cpacs element. https://www.cpacs.de/documentation/CPACS_3_3_0_Docs/html/89b6a288-0944-bd56-a1ef-8d3c8e48ad95.htm
Should we maybe point that out? I'm not sure, is it equal to "global coordinates" in tigl?

Reply:
Sorry for keeping this open so long. I would still argue that "in CPACS-coordinates" is a slightly confusing formulation. Usually, the preposition "in" is followed by a unit (meters, yards...). Here, this is a relative translation from the wing coordinate system. This corresponds to the translation part of the sections transformation. The units of this translation are implied.